### ISSUE TITLE
Hydrostatic Box Simulation root convergence error

### ISSUE THREAD

**Issue by LanaLubecke** [CliMA/ClimaAtmos.jl#1954]
The hydrostatic box simulations runs correctly using the test written in the pipeline:

`      - label: ":computer: Box hydrostatic balance (ρe_tot)"
        command: >
          julia --color=yes --project=examples examples/hybrid/driver.jl
          --config box --perturb_initstate false --hyperdiff false
          --initial_condition DecayingProfile
          --dt 20secs --t_end 1days
          --job_id box_hydrostatic_balance_rhoe
        artifact_paths: "box_hydrostatic_balance_rhoe/*"`

However, when changing the dimensions and resolution of the box, with the following test:

`    julia --color=yes --project=$CA_EXAMPLE $DRIVER \
    --config box --perturb_initstate false  --vert_diff true --C_E 0.044 \
    --initial_condition DecayingProfile \
    --x_max 1e4 --y_max 1e4 --x_elem 6 --y_elem 6 --z_max 3e3 --z_elem 60  \
    --dt 0.5secs --t_end 6hours --dt_save_to_disk 600secs \
    --job_id box_hydrostatic_balance_rhoe --output_dir $OUTPUT_DIR \`

The following error is thrown: LoadError: h_bottom root failed to converge for dz_bottom: 500.0 on domain (0.0, 3000.0).
This post is meant to document this observation in order to see if its a broader issue that needs to be addressed.

**Comment by akshaysridhar** [comment]
I think there is a default command line argument applied here: `--z_stretch true`. This requires the specification of `dz_bottom` and `dz_top` such that, in typical atmosphere simulations, the mesh intervals "stretch" between the minimum cell size near the base, to the maximum cell size at the top of the domain. Default `dz_bottom`,`dz_top` are `500m,5000m` respectively, which is sensible for a simulation of the earth with model top at 30000m (given 60 vertical levels), but seems to result in the convergence error shown above. Does this help? 

In the ClimaAtmos code: `./src/solver/type_getters.jl`
```
146        z_stretch = if parsed_args["z_stretch"]
147          Meshes.GeneralizedExponentialStretching(dz_bottom, dz_top)
```

**Comment by szy21** [comment]
Yes, you need to set `--z_stretch false` to run the resolution you have.
